model_analyzer/config/generate/search_parameters.py

```diff
@@ -120,11 +122,15 @@ def _populate_search_parameters(self) -> None:
 
     def _populate_parameters(self) -> None:
         self._populate_batch_sizes()
-        self._populate_concurrency()
-        # TODO: Populate request rate - TMA-1903
+
+        if not self._is_bls_model:
```

Comment:
where is concurrency populated when is it a bls model?

Reply:
It should be populated for the top-level model only (as concurrency is just specified once as an arg to PA). I've updated the code to reflect this.

Reply:
thanks for explaining that.